src/Microsoft.VisualBasic/src/MatchingRefApiCompatBaseline.netstandard.txt

```diff
@@ -0,0 +1,41 @@
+Compat issues with assembly Microsoft.VisualBasic:
```

Comment:
@jaredpar who would be a good person to determine whether or not we should expose these APIs public in the VB reference assembly?